include/seccomp.h.in

```diff
@@ -198,44 +198,108 @@ struct scmp_arg_cmp {
  */
 #define SCMP_SYS(x)		(__NR_##x)
 
+/* Helper macros for the 32-bit argument comparison macros */
+/* Expand to the number of arguments given */
+#define SCMP_VA_NUM_ARGS(...)	SCMP_VA_NUM_ARGS_IMPL(__VA_ARGS__,2,1)
+#define SCMP_VA_NUM_ARGS_IMPL(_1,_2,N,...)	N
+
+/* Expand to alternative names based on the number of arguments given. */
```

Comment:
I'd drop this comment too and move this block up so there is no vertical whitespace between the SCMP_VA_* macros and the SCMP_MACRO_DISPATCHER_* macros.